### PR TITLE
APP-271: Error after configuring Intel Edison board

### PR DESCRIPTION
**Pull request by Dubland**
/cc @vadimol @RostakaGmfun

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 1]
```diff
@@ -81,6 +81,21 @@ Refer to [the official Edison getting started guide](https://software.intel.com/
...
+7. Create .profile file and set LD_LIBRARY_PATH.
+
+   Add a profile file.
+   ```
+   nano .profile
+   ```
+   Write the following line into the added profile file and then save the file.
+   ```
+   export LD_LIBRARY_PATH=/usr/local/lib
+   ```
```
Comment (vadimol) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: _export LD_LIBRARY_PATH=/usr/local/lib_
will be enough to compile the application. It is not necessary to change user's profile.

Comment (rasendubi) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: note the user could already have a custom `LD_LIBRARY_PATH`, so we need to append here, not replace.

Comment (rasendubi) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: @vadimol that's for running application, not compiling.

Comment (vadimol) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: @rasendubi , I didn't mean the "compilation phase". We run avrocpp during the building of the demo and according to logs in http://jira.kaaproject.org/browse/APP-271, it cannot find run-time libraries.
Yes, of course, we also need run-time libraries to run the application.

Comment (rasendubi) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: oh, sorry. haven't read the issue :unamused:

Comment (Dubland) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Changed

**Review by vadimol — APPROVED** [review]
LGTM

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 2]
```diff
@@ -85,6 +85,7 @@ Refer to [the official Edison getting started guide](https://software.intel.com/
 
 Creating application based on C++ SDK for Linux platform is identical for Intel Edison platform.
 You can follow [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux#quick-way-to-build-sdk).
+Before building your application put this command `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib` in a console for setting directories where libraries should be searched for first.
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Prepend, don't append.

Also, enclose in the `"`.

**Review by rasendubi — CHANGES_REQUESTED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 2 above)
Comment (Dubland) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Changed

**Review by rasendubi — CHANGES_REQUESTED** [review]
You haven't tested the guide. The `"LD_LIBRARY_PATH"` is a normal string, not variable. Use `"$LD_LIBRARY_PATH"`.

**Review by rasendubi — APPROVED** [review]

**Review by vadimol — APPROVED** [review]
LGTM

**Pull request merged**